### PR TITLE
FARSI sim pull request

### PR DESCRIPTION
**Pull request by varun-k-s**
Updated with working training scripts for FARSI simulator.

### CODE REVIEW

**Review by ShvetankPrakash — APPROVED** [review]
LGTM

File: acme/setup.py [hunk 1]
```diff
@@ -58,10 +58,10 @@
...
 jax_requirements = [
     'chex',
-    'jax==0.3.6',  # Update when TF2.9 is release.
-    'jaxlib==0.3.15',  # Update when TF2.9 is release.
-    'dm-haiku',
-    'flax',
+    'jax==0.4.1',  # Update when TF2.9 is release.
+    'jaxlib==0.4.1',  # Update when TF2.9 is release.
+    'dm-haiku==0.0.10',
+    #'flax',
```
Comment (ShvetankPrakash) [inline acme/setup.py:64]: @varun-k-s why did you have to comment out this package?

**Comment by ShvetankPrakash** [comment]
@varun-k-s can you resolve this conflict and then I can merge and close your PR :)

**Review by ShvetankPrakash — APPROVED** [review]

**Comment by ShvetankPrakash** [comment]
@nx6xe23  I approved your latest commit. Can you resolve the merge conflict and then I can close this issue?

**Review by ShvetankPrakash — APPROVED** [review]

File: acme/setup.py (hunk 1 above)
Comment (varun-k-s) [reply acme/setup.py]: I got an "error: subprocess-exited-with-error" when including flax as a package in acme/setup.py.
flax was already being installed as a requirement for some other packages, so commenting the line out did not cause an issue.

Comment (ShvetankPrakash) [reply acme/setup.py]: And assuming everything still worked fine @varun-k-s right?

Comment (varun-k-s) [reply acme/setup.py]: Yes everything works fine

Comment (ShvetankPrakash) [reply acme/setup.py]: Amazing! If you could test that it all works on main now that it has been merged that would be a huge help as well to just confirm all your hard work is officially complete and working for folks to use :) Then I can ping folks who have been interested in using your work!

Comment (ShvetankPrakash) [reply acme/setup.py]: @varun-k-s

**Pull request merged**